src-tauri/src/sql.rs

```diff
@@ -93,7 +93,7 @@ impl DB {
             month,
             year
         )
-        .execute(&mut tx)
+        .execute(&mut *tx)
```

Comment:
Dereference now required

From the changelog:

----

The Executor impls for Transaction and PoolConnection have been deleted because they cannot exist in the new crate architecture without rewriting the Executor trait entirely.
  - To fix this breakage, simply add a dereference where an impl Executor is expected, as they both dereference to the inner connection type which will still implement it:
    - &mut transaction -> &mut *transaction
    - &mut connection -> &mut *connection
  - These cannot be blanket impls as it triggers an overflow in the compiler due to the lack of lazy normalization, and the driver crates cannot provide their own impls due to the orphan rule.
  - We're expecting to do another major refactor of traits to incorporate generic associated types (GAT). This will mean another major release of SQLx but ideally most API usage will not need to change significantly, if at all.